server/etcdserver/cindex/cindex.go

```diff
@@ -106,18 +148,24 @@ type fakeConsistentIndex struct {
 	term  uint64
 }
 
-func (f *fakeConsistentIndex) ConsistentIndex() uint64 { return f.index }
+func (f *fakeConsistentIndex) ConsistentIndex() uint64                   { return f.index }
```

Comment:
nit: If we are storing as atomic, we should follow the pattern and load (`atomic.LoadUint64`) as atomic as well.

Reply:
thanks & done.